src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -607,8 +615,15 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
       vrMode,
       vrFileExists,
       multiImages,
+      detailsProductData,
     } = this.state;
 
+    const productDescription = detailsProductData ? detailsProductData[0]['display-value'].split('. ', 1)[0] : '';
```

Comment:
Have you seen a case where `display-value` is undefined or null? Could an error get thrown here?